test/js/setup-tests.js

```diff
@@ -24,6 +24,10 @@ global.scrollTo = jest.fn()
 global.TextEncoder = TextEncoder
 global.TextDecoder = TextDecoder
 
+global.SCP = {
```

Comment:
Can you add a comment as to why this is necessary?  Seeing this sprinkled across the codebase triggers my 'code-smell' warnings...

Reply:
Good suggestion, done in 52108d8.  Those checks sprinkled through the codebase were previously rather implicit / opaque ways to detect if the code was running in a test.  This makes those checks more explicit and clearer.